### ISSUE TITLE
add yant jouet

### ISSUE THREAD

**Issue by kintaro1981** [maxcleme/beadcolors#48]
I would like to add yantjouet beads colors.

The color table is available on the description of their products, example: https://www.aliexpress.com/item/32996983940.html?#nav-description
Go to description and then view more

What I have to do to add them to beadifier?
Take RGB info from those images or I need to take RGB info from the real beads?

Regards

**Comment by maxcleme** [comment]
@kintaro1981 Do you mind posting the "color table" here? Looks like the product isn't available in my region so I'm struggling to find what you're speaking of.

**Comment by kintaro1981** [comment]
![yantjouet_2 6](https://github.com/user-attachments/assets/e41a0c36-241e-423c-9f75-dd8e2c3a6e81)
this is an example from the description mentionet in my first post.
I'm not sure if this the whole table and if the colors are the same for 2.6mm and 5mm beads

**Comment by kintaro1981** [comment]
Exactly, it was my initial question.
Now, as I have some of them how I calculate RGB? Do I need expensive device like [this](https://www.reddit.com/r/beadsprites/comments/csrppb/scientific_artkal_perler_rgb_colors_the_bead/)?

**Comment by maxcleme** [comment]
Kind of! Or maybe the manufacturer can give you those references? Did you try to contact them directly?

**Comment by maxcleme** [comment]
RBG components is what matters yes

**Comment by kintaro1981** [comment]
They sent me the colors table for their 5mm version beads.
I have their beads in 5mm and 2.6mm versions and I think the colors are the same (but I'm going to check more accurately).

This is the colors board for the 5mm
![yant_beads_colors_5mm](https://github.com/user-attachments/assets/d59df197-b72e-4e9e-afaa-4a8464686cd4)

I think I can go ahead and create the raw file, right?

**Comment by maxcleme** [comment]
> I think I can go ahead and create the raw file, right?

Definitely! I think we should start having the same file for both 2.6mm and 5mm, we could expand the 2.6mm later then! 😁 

Great find!

**Comment by kintaro1981** [comment]
I have to duplicate the CSV for minis?

**Comment by maxcleme** [comment]
> Hve I to duplicate the CSV for minis?

I guess we could symlink the file, this is handled by GitHub IIRC, but tbh I don't mind copy/pasting the file 👍

**Comment by kintaro1981** [comment]
Let me now if I need to do another step or simply wait patiently.
;-)

**Comment by maxcleme** [comment]
Thanks again for your contribution! I can confirm the new palette is live on [beadifier](https://beadifier.eremes.xyz/), you might have to force refresh your browser cache if it doesn't show up!
